vicon/udp.h

```diff
@@ -171,6 +173,47 @@ template<typename ObjectDataType = ObjectData>
 class UDPClient
 {
 public:
+    class Object
```

Comment:
I think this is a nice idea but maybe call it ``ObjectReference`` or ``ObjectRef`` or something to save confusion? Not totally sure it needs to be a child class of ``UDPClient`` either.

Reply:
Yeah good point. It's not a great name.